### DISCUSSION TITLE
Validate Max Rule

### DISCUSSION THREAD

**Discussion by AndrewMarkUK** [dedoc/scramble#241] [Q&A]
I have a validate rule that looks like this:

'action_name' => 'required|string|max:50'

In the API Documentation then, it is displayed as required and as a string BUT...there is no display for max characters.
![Screenshot 2023-09-27 181533](https://github.com/dedoc/scramble/assets/8135582/0d956b3b-d324-4723-9395-aa7a0d35e729)

I am trying to tell tell the user in the docs that action_name can have a max character count of 50.

Is there something I am missing here?

**Comment by romalytvynenko** [comment]
@AndrewMarkUK Hey! You are not missing anything, this is not yet implemented 🫣

**Reply by AndrewMarkUK** [reply]
I look forward to when thse additional rules can be implemented. :-)
